### PR TITLE
Make rename functions accessibly by keyboard navigation

### LINKED ISSUE #3797: Cannot rename card with keyboard navigation

### PR DESCRIPTION
**Pull request by juliushaertl**
Fixes #3797 

https://user-images.githubusercontent.com/3404133/168068637-353ac21a-43ad-4f63-b339-f196ac34ebe3.mov

### CODE REVIEW

**Review by nimishavijay — APPROVED** [review]
Looks really nice!

File: src/components/cards/CardItem.vue [hunk 1]
```diff
@@ -38,11 +38,15 @@
 				<h3 v-if="compactMode || isArchived || showArchived || !canEdit || standalone">
```
Comment (vinicius73) [inline src/components/cards/CardItem.vue:38]: Can we move it to a computed?

Comment (juliushaertl) [reply src/components/cards/CardItem.vue]: Yes, moved :)

File: src/components/cards/CardItem.vue [hunk 2]
```diff
@@ -135,6 +139,9 @@ export default {
 			const board = this.$store.getters.boards.find((item) => item.id === this.card.boardId)
 			return board ? !board.archived && board.permissions.PERMISSION_EDIT : false
 		},
+		inlineEditingBlocked() {
+			return compactMode || isArchived || showArchived || !canEdit || standalone
```
Comment (luka-nextcloud) [inline src/components/cards/CardItem.vue:143]: I think `compactMode`, `isArchived`, `showArchived`, `canEdit`, `standalone` variables are undefined as you can see in the github action.

Comment (juliushaertl) [reply src/components/cards/CardItem.vue]: Ah, good catch

**Comment by juliushaertl** [comment]
@vinicius73 @luka-nextcloud Reviews would be highly appreciated :)

**Pull request merged**